backend/testHelper/spotifyServer.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
```

Comment:
maybe change the folder structure from `backend/testHelper/`... to `backend/test/helper` for a more readable code.